src/rulesets/oas/index.json

```diff
@@ -198,21 +198,6 @@
         }
       ]
     },
-    "only-local-references": {
```

Comment:
Technically this is a breaking change.

Reply:
SemVer only discusses breaking changes to the API, not removing a result from a collection. Consider the use case here, an unrecommended rule has been forcibly enabled by somebody, and it’s immediately started breaking things for anyone who actually has a remote $ref, so they disable it. 

I think this rule is currently a big which should have been removed in 4.0 when we added support for remote refs. If somebody somewhere has enabled this and do want it they can add it back, right?

Reply:
You are absolutely right.

Reply:
> SemVer only discusses breaking changes to the API

If we consider the API is only the documented stuff then yes, that's all correct. But I disagree with such definition. API is everything that is **public** (as in the `public` keyword in Java/TypeScript). From a technical point of view it doesn't matter something is not documented. If by removing something that is `public`, but was not documented, (consider situation where a codebase simply doesn't have any docs) we _may_ break someone's code then it's a breaking change. And that's what semantic versioning talks about. They don't talk about documentation afaik.

Reply:
@chris-miaskowski the API is the interface, not the result. We are changing the result not the interface. Nothing to do with documentation :)

Reply:
Oooh, ok, true. I didn't see it's that generated json file. Sorry ;)